ale_linters/haskell/cabal_v2_ghc.vim

```diff
@@ -0,0 +1,20 @@
+" Author: Eric Wolf <ericwolf42@gmail.com>
+" Description: ghc for Haskell files called with cabal exec
+
+call ale#Set('haskell_cabal_ghc_options', '-fno-code -v0')
```

Comment:
I think you can cut this file out for now. We could add an `executable` option later for switching the cabal linter to `cabal_v2_ghc`.